Common/ColorConv.cpp

```diff
@@ -643,7 +643,7 @@ Convert16bppTo16bppFunc ConvertRGB565ToBGR565 = &ConvertRGB565ToBGR565Basic;
 #endif
 
 void SetupColorConv() {
-#if defined(HAVE_ARMV7) && !defined(ARM64)
+#if PPSSPP_ARCH(ARMV7) && PPSSPP_ARCH(ARM_NEON)
```

Comment:
Yes, later. I'm trying to get it to build, get the basic review done, then we can incrementally fix all of those.